scripts/extract-release-notes.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument("out_path")
+args = parser.parse_args()
+
+with open("CHANGELOG.md") as f:
+    changelog = f.read()
+
+marker = "\n## "
+
+idx_unreleased = changelog.find(marker)
```

Comment:
This seems error prone. If we rename `Unreleased` to `v1.2.3` in a commit before we make the tag, this will pull the correct section. But if we keep an `Unreleased` section before that, it won't.

Reply:
My assumption was that the first version is always Unreleased, and then the newest version, which you're about to release. As long as we decide on a convention it should be fine, but I'll also add the suggestion by @jumaffre to validate that the version matches.

Reply:
I've rewritten the script, it now searches for the git version number.